frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -131,22 +132,15 @@ const RiderModalInfo = ({
           )}
         </div>
         <div className={cn(styles.gridR2, styles.gridCBig1)}>
-          <Label className={styles.label} htmlFor="needs">
-            Needs:{' '}
-          </Label>
-          <select
-            name="needs"
-            aria-required="true"
-            ref={register({ required: true })}
-          >
-            {Object.values(Accessibility).map((value, index) => {
-              return (
-                <option key={index} value={value}>
-                  {value}
-                </option>
-              );
-            })}
-          </select>
+          Needs:
+          <DropdownBox
+            placeHolderText="Select Your Needs"
+            dataSource={Object.values(Accessibility)
+              .splice(1)
```

Comment:
instead of using the splice we can just remove the None value from the Accessibility enum since the stakeholders suggested there is no need for it to be there